build/scripts/test.ps1

```diff
@@ -0,0 +1,36 @@
+# Script for testing out the various functions on a given machine.  Useful for
```

Comment:
Call the script test-machine-setup or something? People might think this runs our tests...